app/src/views/SingleTeamView.vue

```diff
@@ -47,14 +47,7 @@
         @toggleAddMemberModal="showAddMemberForm = !showAddMemberForm"
       />
     </div>
-    <TipsAction
-      :addresses="team.members.map((member) => member.walletAddress)"
-      :pushTipLoading="pushTipLoading"
-      :sendTipLoading="sendTipLoading"
-      :tipAmount="tipAmount"
-      @pushTip="(addresses, amount) => pushTip(addresses, amount)"
-      @sendTip="(addresses, amount) => sendTip(addresses, amount)"
-    />
+    <TipsAction :addresses="team.members.map((member) => member.walletAddress)" />
```

Comment:
yes, i done it because the loading will not working if we define the function in the parent and get the loading in children component. but i think it's possible to do it if we pass the loading from parent as a props, but in my opinion it makes in the component will be complex (IMO).

we dont have to use "emit events" since we already seperate the logic into the composables